Modules/EMCAL/src/CalibCheck.cxx

```diff
@@ -93,6 +93,15 @@ void CalibCheck::configure()
     }
   }
 
+  auto nBadThresholdChannelsFEC = mCustomParameters.find("BadThresholdChannelsFEC");
+  if (nBadThresholdChannelsFEC != mCustomParameters.end()) {
+    try {
+      mBadThresholdChannelsFEC = std::stof(nBadThresholdChannelsFEC->second);
+    } catch (std::exception& e) {
+      ILOG(Error, Support) << fmt::format("Value {} not a float", nBadThresholdChannelsFEC->second.data()) << ENDM;
```

Comment:
As mentioned in the other PR the format library is not needed. Please go through the loggings and fix accordingly.